.github/workflows/build-and-test.yml

```diff
@@ -25,4 +25,4 @@ jobs:
       - name: Test code generation
         run: npm run generate
       - name: Testing
-        run: npm run test
+        run: npm run test-ci
```

Comment:
we can probably remove test-ci and simply add test and coverage as separated scripts, this should be more clear.